modules/cnv-uploading-local-disk-image-dv.adoc

```diff
@@ -16,31 +16,41 @@ compressed by using `xz` or `gz`.
 * The `kubevirt-virtctl` package must be installed on the client machine.
 * The client machine must be configured to trust the {product-title} router's
 certificate.
-* A spare DataVolume that is the same size or larger than the disk that you are uploading.
 
 .Procedure
 
 . Identify the following items:
-** File location of the VM disk image that you want to upload
-** Name of the DataVolume
-
-. Run the `virtctl image-upload` command to upload your disk image.
-You must specify the DV name and file location. For example:
+** The name of the upload DataVolume that you want to use. If this DataVolume
+does not exist, it is created automatically.
+** The size of the DataVolume, if you want it to be created during the upload
+procedure. The size must be greater than or equal to the size of the disk image.
+** The file location of the virtual machine disk image that you want to upload.
+
+. Upload the disk image by running the `virtctl image-upload` command.
+Specify the parameters that you identified in the previous step.
+For example:
 +
 ----
-$ virtctl image-upload --dv-name=<upload-datavolume> --image-path=</images/fedora.qcow2> <1> <2>
+$ virtctl image-upload dv <datavolume_name> \ <1>
+--size=<datavolume_size> \ <2>
+--image-path=</path/to/image> \ <3>
 ----
-<1> The name of the DataVolume that you are creating.
-<2> The filepath of the virtual machine disk image you are uploading.
+<1> The name of the DataVolume.
+<2> The size of the DataVolume. For example: `--size=500Mi`, `--size=1G`
+<3> The file path of the virtual machine disk image.
 +
-[CAUTION]
+[NOTE]
 ====
-To allow insecure server connections when using HTTPS, use the `--insecure`
+* If you do not want to create a new DataVolume, omit the `--size` parameter and
+include the `--no-create` flag.
+
+* To allow insecure server connections when using HTTPS, use the `--insecure`
 parameter. Be aware that when you use the `--insecure` flag, the authenticity of
 the upload endpoint is *not* verified.
 ====
 
-. To verify that the DV was created, view all DV objects:
+. Optional. To verify that a DataVolume was created, view all DV objects by running the
```

Comment:
Mix of DataVolume and DV makes this a bit odd to parse. I understand why it's like that, and not suggesting a change, but wanted to flag it just in case.